tests/run-make/multiple-emits/rmake.rs

```diff
@@ -0,0 +1,13 @@
+use run_make_support::{cwd, rustc};
+
+fn main() {
+    rustc().input("foo.rs").emit("asm,llvm-ir").output(cwd().join("out")).run();
+
+    assert!(cwd().join("out.ll").is_file());
```

Comment:
nit: `Path::new("out.ll").is_file()` is perhaps a bit clearer, it is IMO a bit head-scratching when we join paths to the current working directory, since relative paths also work.

Maybe we can add a `is_file` function to `run-make-support`, it will probably be useful also for other tests. Or even just something like a `fn path(path: &str) -> &Path { Path::new(path) }` (again in `run-make-support`) would make this a bit more readable.